bcipy/task/paradigm/rsvp/copy_phrase.py

```diff
@@ -128,7 +128,7 @@ class RSVPCopyPhraseTask(Task):
         "max_inq_per_series",
         "max_minutes",
         "max_selections",
-        "min_inq_len",
+        "min_inq_per_series",
         "show_feedback",
```

Comment:
This came up in a discussion in February and I created a different ticket for it: https://www.pivotaltracker.com/n/projects/2460065/stories/187147945.

Reply:
We did duplicate this! We went with a different solution, but it still addresses the need for a single parameter.